### DISCUSSION TITLE
Character menu question

### DISCUSSION THREAD

**Discussion by TrollerCoasterGR** [Lurkars/gloomhavensecretariat#448] [Q&A]
When we open our character menus during a scenario we noticed 2 icons, an ongoing effect icon and a block icon. What are these for?

**Comment by Lurkars** [comment]
You can create permanent conditions or immunities for that character (also for monsters).

**Reply by Lurkars** [reply]
Here is the corresponding issue, but contains spoiler for one(some?) class:
https://github.com/Lurkars/gloomhavensecretariat/issues/355
